<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="homeAction"/>
    <addaction name="newAction"/>
    <addaction name="openAction"/>
    <addaction name="saveAction"/>
    <addaction name="saveAsAction"/>
    <addaction name="exportAction"/>
    <addaction name="separator"/>
    <addaction name="quitAction"/>
   </widget>
   <widget class="QMenu" name="editMenu">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="undoAction"/>
    <addaction name="redoAction"/>
    <addaction name="cutAction"/>
    <addaction name="copyAction"/>
    <addaction name="pasteAction"/>
    <addaction name="selectAllAction"/>
    <addaction name="deleteAction"/>
   </widget>
   <widget class="QMenu" name="viewMenu">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="separator"/>
    <addaction name="resetViewAction"/>
    <addaction name="showGridAction"/>
    <addaction name="zoomInAction"/>
    <addaction name="zoomOutAction"/>
    <addaction name="fullScreenAction"/>
   </widget>
   <widget class="QMenu" name="helpMenu">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="documentationAction"/>
    <addaction name="aboutAction"/>
    <addaction name="reportAction"/>
    <addaction name="contactAction"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="moveToolAction"/>
    <addaction name="rotateToolAction"/>
    <addaction name="scaleToolAction"/>
    <addaction name="alignObjectsAction"/>
    <addaction name="threeDViewAction"/>
    <addaction name="resetPositionAction"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="toggleSidebarAction"/>
    <addaction name="toggleConsoleAction"/>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="editMenu"/>
   <addaction name="viewMenu"/>
   <addaction name="menuTools"/>
   <addaction name="menuWindow"/>
   <addaction name="helpMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="resetViewAction">
   <property name="icon">
    <iconset theme="system-reboot"/>
   </property>
   <property name="text">
    <string>Reset view</string>
   </property>
  </action>
  <action name="showGridAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show grid</string>
   </property>
  </action>
  <action name="homeAction">
   <property name="text">
    <string>Home</string>
   </property>
  </action>
  <action name="openAction">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="saveAction">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="saveAsAction">
   <property name="text">
    <string>Save as</string>
   </property>
  </action>
  <action name="exportAction">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="quitAction">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="undoAction">
   <property name="icon">
    <iconset theme="edit-undo"/>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="redoAction">
   <property name="icon">
    <iconset theme="edit-redo"/>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="cutAction">
   <property name="icon">
    <iconset theme="edit-cut"/>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="copyAction">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditCopy"/>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="pasteAction">
   <property name="icon">
    <iconset theme="edit-paste"/>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="deleteAction">
   <property name="icon">
    <iconset theme="edit-delete"/>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="selectAllAction">
   <property name="icon">
    <iconset theme="edit-select-all"/>
   </property>
   <property name="text">
    <string>Select All</string>
   </property>
  </action>
  <action name="zoomInAction">
   <property name="icon">
    <iconset theme="zoom-in"/>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
  </action>
  <action name="zoomOutAction">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::ZoomOut"/>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="fullScreenAction">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::ZoomFitBest"/>
   </property>
   <property name="text">
    <string>Full Screen</string>
   </property>
  </action>
  <action name="moveToolAction">
   <property name="text">
    <string>Move tool</string>
   </property>
  </action>
  <action name="rotateToolAction">
   <property name="text">
    <string>Rotate tool</string>
   </property>
  </action>
  <action name="scaleToolAction">
   <property name="text">
    <string>Scale tool</string>
   </property>
  </action>
  <action name="alignObjectsAction">
   <property name="text">
    <string>Align Objects</string>
   </property>
  </action>
  <action name="threeDViewAction">
   <property name="text">
    <string>3D View</string>
   </property>
  </action>
  <action name="resetPositionAction">
   <property name="text">
    <string>Reset Position</string>
   </property>
  </action>
  <action name="toggleSidebarAction">
   <property name="text">
    <string>Toggle Sidebar</string>
   </property>
  </action>
  <action name="toggleConsoleAction">
   <property name="text">
    <string>Toggle Console</string>
   </property>
  </action>
  <action name="documentationAction">
   <property name="text">
    <string>Documentation</string>
   </property>
  </action>
  <action name="aboutAction">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="reportAction">
   <property name="text">
    <string>Report</string>
   </property>
  </action>
  <action name="contactAction">
   <property name="text">
    <string>Contact</string>
   </property>
  </action>
  <action name="newAction">
   <property name="text">
    <string>New</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
